sgf-viewer/src/content.ts

```diff
@@ -184,10 +184,28 @@ export const pages: object =
                 ],
                 "max_games": 16,
                 "adversary": "498 million training steps, 600 visits",
-                "victim": "<code>Latest</code>, 1600 visits",
+                "victim": "<code>Latest</code><sub><code>def</code></sub>, 1600 visits",
                 "description": [
                     "We make available here the full game records for the positions analyzed with different levels of visits in the <a target=\"_blank\" href=\"/pdfs/go_attack_paper.pdf#appendix.H\">paper appendix discussing the role of search in robustness</a>. For details, please refer to the appendix."
                 ]
+            },
+            {
+                "title": "Positions analyzed with 1 billion visits",
+                "dir_name": "position-analysis-1b",
+                "server": "dqn.ist.berkeley.edu",
+                "paths": [
+                    "/nas/ucb/kpelrine/position-analysis/1b/17C295C81D606485.sgf",
+                    "/nas/ucb/kpelrine/position-analysis/1b/410B1AA59B8501F3.sgf",
+                    "/nas/ucb/kpelrine/position-analysis/1b/517ACE2F2CF6A19C.sgf",
+                    "/nas/ucb/kpelrine/position-analysis/1b/6E4A22DAD4680325.sgf",
+                    "/nas/ucb/kpelrine/position-analysis/1b/9E85957D1FD7E400.sgf"
+                ],
+                "max_games": 5,
+                "adversary": "545 million training steps, 600 visits",
+                "victim": "<code>Latest</code>, 1 million visits",
```

Comment:
Good question, it's a really complicated position and wasn't obvious for me either. Black would like to kill the white group in the middle of the left side, but attacking it immediately won't work. For example, after the sequence below, black is in atari and has to capture above 8, but then white retakes the ko and black has no ko threats large enough to win.

<img width="996" alt="image" src="https://user-images.githubusercontent.com/46732724/218896940-0982377b-8c35-4afa-8399-2794ae9e12ab.png">

So the goal of corner first is to fill up a few of white's liberties there, so that when black attacks the side group white has to make more liberties for their own corner before filling all of black's liberties. 

<img width="998" alt="image" src="https://user-images.githubusercontent.com/46732724/218895375-65cd548b-a4d6-4c37-b49a-a2a10a498605.png">

Something like the above. At this point, white would like to fill in black's last liberties, and if the corner didn't have any issues white could, but as things stand to play B16 white needs to capture the black stones in the corner first. There are a whole bunch of other variations, for instance black doesn't have to play 3 here immediately, but this one is a sure way for black to escape.

"X" in the original figure doesn't work because white will play at "✓" themselves, fixing their corner problems, and "X" didn't get any new liberties so the rest of the situation remains the same.

Reply:
> it's a tiny bit confusing that the victim is 1 million visits and the description says 1 billion visits. But it's explained in the paper and I guess this subsection doesn't really make sense without looking at the paper anyway

True, I'll add an additional line in the text to clarify.